plugin.go

```diff
@@ -40,7 +42,7 @@ func (p Plugin) uploadCommand() *exec.Cmd {
 	args = append(args, p.Username)
 	args = append(args, "--password")
 	args = append(args, p.Password)
-	args = append(args, "dist/*")
+	args = append(args, filepath.Join(filepath.Dir(p.DistDir), "/*"))
```

Comment:
you can simplify to this:

```
filepath.Join(p.DistDir, "/*")
```

Reply:
@bradrydzewski Thanks! Pushed this change.